PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellEnergy.cxx

```diff
@@ -232,10 +229,10 @@ Int_t AliEmcalCorrectionCellEnergy::InitRunDepRecalib()
   // opening the OADB container
   // 
   // For more information see https://alice.its.cern.ch/jira/browse/EMCAL-135
-  if(fEventManager.InputEvent()->GetRunNumber() > 197692){
-    AliError("Temperature recalibration disabled for run2");
-    return 0;
-  }
+  //if(fEventManager.InputEvent()->GetRunNumber() > 197692){
```

Comment:
Since we resolved the issue and are removing this file, can we fully remove it and the comment? (I guess this doesn't need to be in this PR, but I worry that it will be forgotten otherwise)

Reply:
Yes, It will be removed completely.

Reply:
Great, thanks!

Reply:
Whole block including description removed.

Reply:
Thanks!